### PR TITLE
Release/1.11.18

### PR DESCRIPTION
**Pull request by AxelDelmas**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
   },
   "dependencies": {
     "codem-isoboxer": "0.2.2",
-    "dashjs": "2.4.1",
+    "dashjs": "streamroot/dash.js#patches-build",
```
Comment (jlouazel) [inline package.json]: Is it really safe to depend on a fork that won't be maintained?

Comment (AxelDelmas) [inline package.json]: We relied on a fixed version of dash.js, it does not change anything.
We'll change it back to an official dash.js version once the fix we need have been merged

Comment (jlouazel) [inline package.json]: OK then you've my 👍

**Comment by ruslandinov** [comment]
Please update the wrapper's `README` about bundled dash.js version saying that for now, temporarily , it's a forked version, with an explanation why we did it.

**Comment by AxelDelmas** [comment]
@ruslandinov done

**Pull request merged**